scania-digital-design-system/tegel
# docs(table): added tds-table-body to the filtering and pagination

**Describe pull-request**  
Adds missing elements to the filtering and pagination default of storybook for table components

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [CDEP-3015](https://tegel.atlassian.net/browse/CDEP-3015)

**How to test**  
_Add description how to test if possible_
1. Go to storybook table filtering/default and pagination/default
2. Navigate to docs page
3. Make sure that `<tds-table-body>` has been added around all the `<tds-table-body-row>`s

**Suggested test steps**

**Screenshots**  
<img width="712" alt="Screenshot 2023-12-05 at 14 18 06" src="https://github.com/scania-digital-design-system/tegel/assets/605500/71bea8d3-1eca-4213-b346-58980823bbfe">


**Additional context**  
noticed this when adding unit tests with "implementation details" :D


[CDEP-3015]: https://tegel.atlassian.net/browse/CDEP-3015?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
